tests/pages/scan.test.js

```diff
@@ -141,7 +141,10 @@ test('Correct error appears when I submit a duplicate voucher', async t => {
         .click(submitButton)
     ;
     const errorMessage = await el('.message').innerText;
-    expect(errorMessage).to.contain('It looks like this code has already been added, please double check and try again. If you are still unable to add the voucher code, don\'t worry - you will still receive payment if you send it in with your other vouchers.');
+    expect(errorMessage).to.contain("It looks like the code (:code) has been used already, please double check and try again. "
+        + "If you are still unable to add the voucher code, don't worry - mark it as \"unrecorded\", "
+        + "send it in with your other vouchers and you will still be paid when we receive it."
```

Comment:
Can we add this t https://trello.com/c/xSHGbdNC/522-change-copy-for-invalid-voucher-code-error-message ? it's a very similar task.